### PR TITLE
Add few more details about the project

### PR DESCRIPTION
**Pull request by georgem3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
 We've applied the philosophy of "Do one thing and do it well" giving you performance and scalability unmatched by any other platform.
 
-Build instructions:
+___
+
+### Key Features:
+* RaiBlocks utilizes a novel block-lattice architecture, unlike conventional blockchains used in many other cryptocurrencies.
+* The network requires minimal resources, no high-power mining hardware, and can process high transaction throughput.
+* Offers feeless, instantaneous transactions, as well as unlimited scalability, making RaiBlocks ideal for peer-to-peer transactions.
+* To date, the RaiBlocks network has processed over four million transactions with an unpruned ledger size of only 1.7GB.
```
Comment (androm3da) [inline README.md]: Please change "To date" to something like "As of December 2017"

Comment (georgem3) [reply README.md]: Fixed!

**Comment by androm3da** [comment]
Please squash these commits.

**Comment by georgem3** [comment]
Like so? Sorry, still getting the hang of git

**Pull request merged**